File: Java/Simple Operations And Calculations/Lab/CircleAreaAndPerimeter.java
import java.util.Scanner;

public class CircleAreaAndPerimeter {
    public static void main(String args[]){
        Scanner scanner = new Scanner(System.in);
        double r = Double.parseDouble(scanner.nextLine());

        double area = r*r*Math.PI;
        double perimeter = 2*r*Math.PI;

        System.out.printf("%.2f", area);
        System.out.println();
        System.out.printf("%.2f", perimeter);
    }
}

File: C#/02. Simple Operations and Calculations/Lab/Circle Area and Perimeter.cs
using System;

namespace Circle_Area_and_Perimeter
{
    class Program
    {
        static void Main(string[] args)
        {
            double r = double.Parse(Console.ReadLine());
            double area = Math.PI * Math.Pow(r, 2);
            double perimeter = 2 * Math.PI * r;

            Console.WriteLine("{0:F2}", area);
            Console.WriteLine("{0:F2}", perimeter);
        }
    }
}
